### ISSUE TITLE
Overflow when using split/half screen mode on Flutter Gallery

### ISSUE THREAD

**Issue by GaryQian** [flutter/flutter#22586]
When enabling the split screen feature on (newer versions of) Android, the icons on Flutter Gallery home page overflows:
![screenshot_20181002-155038](https://user-images.githubusercontent.com/1887398/46381479-5a0e2480-c65b-11e8-97c8-9136c0b226d3.png)
Should the container of the icons allow scrolling?

**Comment by jonahwilliams** [comment]
I don't believe the demo has the correct config to go into split screen outside of a debug build.

**Comment by GaryQian** [comment]
The release versions do not look wrong, but just wanted to file this here in case this is relevant in the future. Not too familiar with the gallery, so not sure if this is a problem or not. Feel free to close if this is not important/an issue.
